static/app/views/replays/detail/breadcrumbs/breadcrumbItem.tsx

```diff
@@ -93,32 +104,39 @@ type CrumbItemProps = {
 
 const CrumbItem = styled(PanelItem)<CrumbItemProps>`
   display: grid;
-  grid-template-columns: max-content max-content auto max-content;
-  align-items: center;
+  grid-template-columns: max-content auto;
+  align-items: flex-start;
   gap: ${space(1)};
   width: 100%;
 
   font-size: ${p => p.theme.fontSizeMedium};
   background: transparent;
-  padding: 0;
-  padding-right: ${space(1)};
+  padding: ${space(1)};
   text-align: left;
-
   border: none;
-  border-bottom: 1px solid ${p => p.theme.innerBorder};
-  ${p => p.isHovered && `background: ${p.theme.surface400};`}
-
-  /* overrides PanelItem css */
-  &:last-child {
-    border-bottom: 1px solid ${p => p.theme.innerBorder};
-  }
+  position: relative;
+  ${p => p.isSelected && `background-color: ${p.theme.purple100};`}
+  ${p => p.isHovered && `background-color: ${p.theme.surface100};`}
+  border-radius: 4px;
 
-  /* Selected state */
-  ::before {
+  &::after {
```

Comment:
Might be helpful to add a comment to make it obvious what the point of these `::after` selectors are doing:
```
/* Draw a vertical line behind the breadcrumb icon. The line connects each row together, but is truncated for the first and last items */
```